### PR TITLE
[New] Add jquery-loading-overlay w/ git auto-update

### LINKED ISSUE #8053: [Request] Add gasparesganga/jquery-loading-overlay
**Library name:** gasparesganga/jquery-loading-overlay
*_Git repository url:https://github.com/gasparesganga/jquery-loading-overlay
*_npm package url(optional):
*_License(s):MIT
*_Official homepage:http://gasparesganga.com/labs/jquery-loading-overlay/
**Wanna say something? Leave message here:
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
close #8053, cc @gasparesganga

Pull request for issue: #8053
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/gasparesganga/jquery-loading-overlay
 * Official website (optional, not the repository): https://gasparesganga.com/labs/jquery-loading-overlay/
 * NPM package url (optional): https://www.npmjs.com/package/gasparesganga-jquery-loading-overlay
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 17
   - Count of stars: 178
   - Count of forks: 57
 * NPM download stats (optional):
   - Downloads in the last day: 127
   - Downloads in the last week: 662
   - Downloads in the last month: 2757

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [X] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/gasparesganga-jquery-loading-overlay/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "url": "git://github.com/gasparesganga/jquery-loading-overlay.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/gasparesganga/jquery-loading-overlay.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "loading*"
```
Comment (maruilian11) [inline ajax/libs/gasparesganga-jquery-loading-overlay/package.json]: Please turn to use `"loadingoverlay*"`. According to https://gasparesganga.com/labs/jquery-loading-overlay/#get-it, we don't need to host `.gif`. And remember to remove the file.

File: ajax/libs/gasparesganga-jquery-loading-overlay/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        ]
+      },
+      {
+        "basePath": "extras",
```
Comment (maruilian11) [inline ajax/libs/gasparesganga-jquery-loading-overlay/package.json]: Please turn to use `"extras/loadingoverlay_progress"` here. Thanks.

File: ajax/libs/gasparesganga-jquery-loading-overlay/package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        "files": [
+          "**/*.js"
```
Comment (maruilian11) [inline ajax/libs/gasparesganga-jquery-loading-overlay/package.json]: Please don't use `**/*` in non-dist folder. Thanks. 
`"loadingoverlay_progress*"` would be bettter.

**Comment by sashberd** [comment]
@maruilian11 Thanks for your feedback. The PR was updated. FYR

**Comment by sufuf3** [comment]
@extend1994 @maruilian11 Please help review this PR, thanks.

**Comment by sashberd** [comment]
@extend1994 @maruilian11 ping

**Comment by gasparesganga** [comment]
Hi there, I am the developer of this library.
I've got it already hosted on another CDN (I know this is definitely **not** the place to mention jsDelivr...) but I would be surely pleased to accomodate your request to have it on CDNJS too.
Could you point me to some documentation on *how to include a library to CDNJS* so that I will be able to take care of it?

File: ajax/libs/gasparesganga-jquery-loading-overlay/package.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "gasparesganga-jquery-loading-overlay",
```
Comment (extend1994) [inline ajax/libs/gasparesganga-jquery-loading-overlay/package.json]: I think we can use `jquery-loading-overlay` which is the same as the repo name.
In this case, please also update commit msg & PR title.
Thanks.

**Comment by extend1994** [comment]
@gasparesganga Sorry for late reply!
Here is documentation: https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#b-pull-request. It includes lots of things, if you have any question, just ask us 😄

**Comment by sashberd** [comment]
@extend1994 The PR, PR message and current PR Info was updated. FYR

**Comment by sashberd** [comment]
@extend1994 You asked to add only gif file or whole versions?

**Comment by extend1994** [comment]
@sashberd Whole version.

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@extend1994  The PR was updated. FYR

**Comment by extend1994** [comment]
The above was made because I misunderstood myself to miss something.

**Comment by sashberd** [comment]
@PeterDaveHello ping

**Pull request merged**